Example/Backpack/UIKit/Tokens/ColorTokensViewController.swift

```diff
@@ -0,0 +1,174 @@
+            verticalStackView.bottomAnchor.constraint(equalTo: scrollView.bottomAnchor, constant: -BPKSpacingLg),
+            verticalStackView.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor, constant: BPKSpacingLg),
+            verticalStackView.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor, constant: BPKSpacingLg),
+            verticalStackView.widthAnchor.constraint(equalTo: scrollView.widthAnchor)
+        ])
+        
+        colorTokens.tokens
+            .map(colorSection(section:))
+            .forEach(verticalStackView.addArrangedSubview)
+    }
+    
+    private func colorSection(section: Tokens.Section) -> UIView {
+        let sectionStackView = UIStackView()
+        sectionStackView.axis = .vertical
+        sectionStackView.spacing = BPKSpacingMd
+        sectionStackView.alignment = .leading
+        sectionStackView.distribution = .fill
+        sectionStackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        let sectionLabel = BPKLabel(fontStyle: .textHeading3)
+        sectionLabel.text = section.name
+        sectionStackView.addArrangedSubview(sectionLabel)
+        section.colors
+            .map(tokenView(colorToken:))
+            .forEach(sectionStackView.addArrangedSubview)
+        
+        return sectionStackView
+    }
+    
+    private func tokenView(colorToken: Tokens.Section.Color) -> UIView {
+        let container = UIView()
+        container.translatesAutoresizingMaskIntoConstraints = false
+
+        let colorView = UIView()
+        colorView.translatesAutoresizingMaskIntoConstraints = false
+        colorView.layer.cornerRadius = BPKSpacingMd
+        colorView.layer.masksToBounds = true
+        colorView.layer.borderWidth = 1
+        colorView.backgroundColor = colorToken.color
+        
+        let colorLabel = BPKLabel(fontStyle: .textLabel1)
+        colorLabel.translatesAutoresizingMaskIntoConstraints = false
+        colorLabel.text = colorToken.name
+        
+        container.addSubview(colorView)
+        container.addSubview(colorLabel)
+        
+        NSLayoutConstraint.activate([
+            colorView.leadingAnchor.constraint(equalTo: container.leadingAnchor, constant: BPKSpacingMd),
+            colorView.widthAnchor.constraint(equalToConstant: BPKSpacingLg),
+            colorView.heightAnchor.constraint(equalToConstant: BPKSpacingLg),
+            colorView.centerYAnchor.constraint(equalTo: container.centerYAnchor),
+            
+            colorLabel.topAnchor.constraint(equalTo: container.topAnchor, constant: BPKSpacingMd),
+            colorLabel.leadingAnchor.constraint(equalTo: colorView.trailingAnchor, constant: BPKSpacingMd),
+            colorLabel.bottomAnchor.constraint(equalTo: container.bottomAnchor, constant: -BPKSpacingMd),
+            colorLabel.trailingAnchor.constraint(equalTo: container.trailingAnchor, constant: -BPKSpacingMd)
```

Comment:
Can we 'flip this around'? So we don't need the `-` on the spacing?

So container.trailingAnchor.constraint(equalTo: colorLabel.trailingAnchor, constant: <spacing>)